models/fileModel.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require("mongoose");
```

Comment:
Why `fileModel.js` is required if file.js has the same schema?

Reply:
removed